snap/info.go

```diff
@@ -65,3 +71,13 @@ type AppInfo struct {
 	Plugs   map[string]*PlugInfo
 	Slots   map[string]*SlotInfo
 }
+
+// StoreInfo provides specific information for a store hosted snap.
+type StoreInfo struct {
+	LastUpdated     time.Time
+	DownloadSha512  string
+	DownloadSize    int64
+	AnonDownloadURL string
+	DownloadURL     string
```

Comment:
Why do we have two download URLs here?  Can we do an emptiness check and get rid of the least privileged one?